Library/ViewModels/CheckoutViewModel.swift

```diff
@@ -438,67 +439,62 @@ private func isNavigationLoadedByWebView(navigation navigation: Navigation?) ->
   }
 }
 
-private func isStripeRequest(request request: NSURLRequest) -> Bool {
-  return request.URL?.host?.hasSuffix("stripe.com") == true
+private func isStripeRequest(request: URLRequest) -> Bool {
+  return request.url?.host?.hasSuffix("stripe.com") == true
 }
 
 private func applePayCheckoutNextJS(forPaymentData paymentData: PaymentData, stripeToken: String)
   -> String? {
 
-  let tokenData = paymentData.tokenData
-
-  let json: [String:[String:String]] = [
-    "apple_pay_token": [
-      "transaction_identifier": tokenData.transactionIdentifier,
-      "payment_network": tokenData.paymentMethodData.network,
-      "payment_instrument_name": tokenData.paymentMethodData.displayName
-      ].compact(),
-    "stripe_token": [
-      "id": stripeToken
-    ]
-  ]
-
-  return (try? NSJSONSerialization.dataWithJSONObject(json, options: []))
-    .flatMap { String(data: $0, encoding: NSUTF8StringEncoding) }
-    .map { json in "window.checkout_apple_pay_next(\(json));" }
+    let tokenData = paymentData.tokenData
+
+    var json: [String:[String:String]] = [:]
+
+    json["apple_pay_token"] = [:]
+    json["apple_pay_token"]?["transaction_identifier"] = tokenData.transactionIdentifier
```

Comment:
nah, was getting weird type errors with it wanting to interpreting values as `Any` instead of `[String:String]` or something...